zhazam.xcodeproj/project.pbxproj

```diff
@@ -52,6 +52,13 @@
 		24A61750244FA9F8007F92E6 /* CountdownViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 24A6174F244FA9F8007F92E6 /* CountdownViewController.swift */; };
 		24A6175B245209A1007F92E6 /* TimerRowView.swift in Sources */ = {isa = PBXBuildFile; fileRef = 24A6175A245209A1007F92E6 /* TimerRowView.swift */; };
 		24A6175D245209AC007F92E6 /* TimerRowView.xib in Resources */ = {isa = PBXBuildFile; fileRef = 24A6175C245209AC007F92E6 /* TimerRowView.xib */; };
+		24C632AF24671E4A00FC0DFA /* LeaderBoardViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 24C632AE24671E4A00FC0DFA /* LeaderBoardViewController.swift */; };
+		24C632B124671E6D00FC0DFA /* LeaderBoardViewController.xib in Resources */ = {isa = PBXBuildFile; fileRef = 24C632B024671E6D00FC0DFA /* LeaderBoardViewController.xib */; };
+		24C632B424671F7D00FC0DFA /* LeaderBoardCell.swift in Sources */ = {isa = PBXBuildFile; fileRef = 24C632B324671F7D00FC0DFA /* LeaderBoardCell.swift */; };
+		24C632B624671F8F00FC0DFA /* LeaderBoardCell.xib in Resources */ = {isa = PBXBuildFile; fileRef = 24C632B524671F8F00FC0DFA /* LeaderBoardCell.xib */; };
+		24C632B92467261F00FC0DFA /* LeaderBoardRowModel.swift in Sources */ = {isa = PBXBuildFile; fileRef = 24C632B82467261F00FC0DFA /* LeaderBoardRowModel.swift */; };
```

Comment:
let's call ViewModel instead of RowModel (also change it for another file you created before, I remember that you have already used this kind of naming)

Reply:
Made corrections